share/completions/mariner.fish

```diff
@@ -0,0 +1,35 @@
+set -l subcommands complete config details download help magnet open search
+
+# Subcommands
+complete -f -n "__fish_use_subcommand $subcommands" -c mariner -a 'complete' --description "Print bash completion command (cliff)"
+complete -f -n "__fish_use_subcommand $subcommands" -c mariner -a 'config' --description "Show or update configuration."
+complete -f -n "__fish_use_subcommand $subcommands" -c mariner -a 'details' --description "Show details about torrent with given ID."
+complete -f -n "__fish_use_subcommand $subcommands" -c mariner -a 'download' --description "Download torrent with given ID."
+complete -f -n "__fish_use_subcommand $subcommands" -c mariner -a 'help' --description "Print detailed help for another command (cliff)"
+complete -f -n "__fish_use_subcommand $subcommands" -c mariner -a 'magnet' --description "Copy magnet link with given ID to clipboard."
+complete -f -n "__fish_use_subcommand $subcommands" -c mariner -a 'open' --description "Open torrent in the default torrent application."
+complete -f -n "__fish_use_subcommand $subcommands" -c mariner -a 'search' --description "Search for torrents."
+
+# Global options
+complete -f -n "__fish_use_subcommand $subcommands" -c mariner -l 'version' --description "Show program's version number and exit"
+complete -f -n "__fish_use_subcommand $subcommands" -c mariner -s 'v' -l 'verbose' --description "Increase verbosity of output. Can be repeated."
```

Comment:
Not all arguments need to be quoted. And while you shouldn't always check if something _has_ to be quoted or not, if it's one simple word it shouldn't be - `-s v -l verbose` is nicer to look at, and saves 4 characters which makes the line a bit shorter.

Also the description should not end in a period (space is at a premium for descriptions because every character less makes it more likely that you can fit more columns).

Reply:
I have a habit to quote all strings, so syntax checker picks them up :]. I've removed them sin the following commit.
Good observation on the periods! I should remove them from Mariner itself as well.